dataset_builders/pie/scientific_papers/scientific_papers.py

```diff
@@ -0,0 +1,81 @@
+import dataclasses
+from typing import Any, Dict, List
+
+import datasets
+from pytorch_ie.core import annotation_field
+from pytorch_ie.documents import TextBasedDocument
+
+from pie_datasets import GeneratorBasedBuilder
+
+
+@dataclasses.dataclass
+class ScientificPapersDocument(TextBasedDocument):
+    """A PIE document for scientific papers dataset."""
+
+    abstract: str = annotation_field()
+    section_names: List[str] = annotation_field()
```

Comment:
`annotation_fields()` requires to be of type `AnnotationLayer[ANNOTATION_TYPE]` instead of `str` where ANNOTATION_TYPE should be subclass of `Annotation`. Please have a look at how `pie_modules.documents.ExtractiveQADocument` (used in the [PIE squad_v2 dataset](https://github.com/ArneBinder/pie-datasets/tree/main/dataset_builders/pie/squad_v2)) is defined, for instance.